tracer/build/_build/GeneratePackageVersions/PackageVersionEntry.cs

```diff
@@ -19,6 +19,8 @@ public class PackageVersionEntry : IPackageVersionEntry
 
         public string MaxVersionExclusive { get; set; }
 
+        public string[] AlwaysIncludeVersions { get; set; } = Array.Empty<string>();
```

Comment:
Maybe add a comment here (as it's not possible in the json) saying why you've added this feature.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2311/commits/91f78e494ba625f39662948538e71029d0cbf93f